src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/processes-procedures/tech-writer-workflow.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Tech Writer Workflow
+contentType: page
+template: basicDoc
+topics:
+  - New Relic only
+  - Tech writer style guide
+  - Processes and procedures
+watermark: NR ONLY
+---
+
+This document will guide you through the entire workflow for editing the New Relic documentation site.
+
+## Resources
+
+- VSCode (or another text editor)
+  - [VSCode has great Github integrations](https://code.visualstudio.com/docs/editor/github)
+- [Github account](https://github.com/join)
+- [Github Desktop](https://desktop.github.com/)
+
+## Setup your local environment
+
+Install Github Desktop, and then navigate to Github Desktop's preferences. On Macs, click on **Github Desktop** in the top left corner of your screen and select **Preferences**. Select the blue **Sign In** button and follow the prompts in the browser window. 
+
+Once Github Desktop is set up, navigate to the [Docs Site repository](https://github.com/newrelic/docs-website) on Github. Click the green **Code** button and then select **Open with Github Desktop**. This will clone the entire repository to your local machine. You can ensure the repo was cloned by navigating to your local Github folder (The default is **~/Documents/Github**). Once you have cloned the repo, you do not need to clone it again in the future.  
+
+## Run the site locally
+
+You will use the Terminal to build the site locally. This will allow you to preview changes before opening a PR. While running, the site will reflect any local changes made. Node and Yarn are tools we use to build the site on your local machine.
+
+### Prerequisites
+- [Install Node](https://nodejs.org/en/download/)
+- Install Yarn 
+  - `npm install -g yarn`
+
+### Build the site
+1. In your Terminal, change your directory `cd` to your cloned repo, **docs-website**. 
+    - `cd ~/Documents/GitHub/docs-website`
+2. Run yarn with the following commands:
+    - `yarn`
+    - `yarn start`
+3. The site will take a few minutes to build. Once it finishes building, you can access your preview site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
+
+## Work on a branch, not a fork
+Some teams work on branches, some teams work on forks; the docs team works in branches. As long as a branch has been pushed upstream, this allows us to work collaboratively and ensure that no work is ever lost when someone goes on FTO.
```

Comment:
"vacation" feels more friendly than "FTO" or maybe I just want it to be a time when something like a vacation feels doable :D